protos/generate.sh

```diff
@@ -1,14 +1,24 @@
 #!/usr/bin/env bash
 set -e
 
-GIT_ROOT=$(git rev-parse --show-toplevel)
-cd "$GIT_ROOT"
+if [[ "$PWD" == '/' ]]; then
```

Comment:
just change the line in dockerfile to  `cd /home/bento/ && ./protos/generate.sh`